src/admin/js/Layout.js

```diff
@@ -0,0 +1,40 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import AppBar from 'material-ui/AppBar';
+import CircularProgress from 'material-ui/CircularProgress';
+import Notification from 'admin-on-rest/lib/mui/layout/Notification';
+import Menu from './Menu';
+
+const Layout = ({ isLoading, children, route }) => {
+    const Title = <Link to="/" style={{ color: '#fff', textDecoration: 'none' }}>{`${APP_NAME} - Administration`}</Link>;
+    const RightElement = isLoading ? <CircularProgress color="#fff" size={0.5} /> : <span />;
+
+    return (
+        <MuiThemeProvider>
+            <div style={{ display: 'flex', flexDirection: 'column', minHeight: '100vh' }}>
+                <AppBar title={Title} iconElementRight={RightElement} />
+                <div className="body" style={{ display: 'flex', flex: '1', backgroundColor: '#edecec' }}>
+                    <Menu />
```

Comment:
You can put the `<Menu>` in the next line to make the content reads first in the source. The `flex` property insures it's displayed first.